src/libcollections/vec.rs

```diff
@@ -456,8 +458,9 @@ impl<T> Vec<T> {
     }
 
     /// Reserves the minimum capacity for exactly `additional` more elements to
-    /// be inserted in the given `Vec<T>`. Does nothing if the capacity is already
-    /// sufficient.
+    /// be inserted in the given `Vec<T>`. After calling `reserve_exact`,
+    /// capacity will be greater than or equal to `self.len() + additional`.
```

Comment:
For what it's worth, here's a relevant comment in the underlying implementation:

https://github.com/rust-lang/rust/blob/b3937ea862e15321e85537ac6bd63ad51099c23f/src/liballoc/raw_vec.rs#L275-L280